docs/recipes/react.md

```diff
@@ -0,0 +1,124 @@
+# Testing React components
+
+## Setting up Babel
+
+The first thing you need to do is to set up `babel` to transpile JSX code from the tests. To do that, I'd recommend the [babelrc recipe](https://github.com/sindresorhus/ava/blob/master/docs/recipes/babelrc.md) using [`babel-preset-react`](http://babeljs.io/docs/plugins/preset-react/). You can also have a look at this [sample project config](https://github.com/adriantoine/ava-enzyme-demo)
+
+## Using [Enzyme](https://github.com/airbnb/enzyme/)
+
+Let's first see how to use AVA with one of the most popular React testing libraries: [Enzyme](https://github.com/airbnb/enzyme).
+
+If you only plan to use `shallow` component rendering, you don't need any extra setup.
```

Comment:
Maybe linkify `shallow component rendering`? Not everyone might know what it means exactly.